core/src/subgraph/instance_manager.rs

```diff
@@ -806,6 +799,24 @@ where
     }
 }
 
+async fn delete_subgraph_firehose_cursor(logger: &Logger, store: &dyn WritableStore) {
+    debug!(logger, "Deleting any existing Firehose cursor");
+    let mut backoff = ExponentialBackoff::new(30 * SECOND, *SUBGRAPH_ERROR_RETRY_CEIL_SECS);
+
+    loop {
+        match store.delete_block_cursor() {
```

Comment:
This is a copy of other existing methods in the trait like `block_ptr`, `block_cursor` and many others. Is the new paradigm to always use `async` from now on?

I would like to perform that in another PR mainly because this is blocking the integration test workflow and it would be nice to unlock it.